src/test/resources/refaster/MethodThrowsRecipe.java

```diff
@@ -85,8 +85,8 @@ public J visitMethodInvocation(J.MethodInvocation elem, ExecutionContext ctx) {
         };
         return Preconditions.check(
                 Preconditions.and(
-                        new UsesType<>("java.nio.file.Files", true),
                         new UsesType<>("java.nio.charset.StandardCharsets", true),
+                        new UsesType<>("java.nio.file.Files", true),
```

Comment:
Not directly related to the changes of this PR, but for `UsesType` I am thinking we can probably set the `includeImplicit` parameter to `false`. What it does is to check the declaring type, return type, and parameter types of each used method, whereas `UsesType` otherwise basically only checks if the source code has some variable or expression of the given type.

It is rather difficult to reason about, so maybe it is safest to leave it as is.

Reply:
Agreed, let's consider this at a later stage.